tests/lib/hw_unique_key_tfm/CMakeLists.txt

```diff
@@ -0,0 +1,34 @@
+#
+# Copyright (c) 2021 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+cmake_minimum_required(VERSION 3.13.1)
+
+if (NOT DEFINED HUK_BOARD_SNR)
```

Comment:
Why is this a CMake variable and not a Kconfig variable that defaults to nothing ?

With Kconfig we have documentation in place automatically.